specification/apimanagement/data-plane/readme.md

```diff
@@ -0,0 +1,58 @@
+# API Management
+    
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Data Plane SDK for API Management.
+
+
+
+---
+## Getting Started 
+To build the SDK for API Management, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information 
+These are the global settings for the API Management API.
+
+``` yaml
+openapi-type: data-plane
+tag: package-2017-03
+```
+
+
+### Tag: package-2017-03
+
+These settings apply only when `--tag=package-2017-03` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-03'
+input-file:
+- Microsoft.ApiManagement/preview/2017-03-01/apimanagement.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimapis.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimauthorizationservers.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimbackends.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimcertificates.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimemailtemplate.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimgroups.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimidentityprovider.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimloggers.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimnetworkstatus.json
```

Comment:
'Microsoft.ApiManagement/preview/2017-03-01/apimnetworkstatus.json' is missing.

Reply:
#Fixed. Good catch.